tests/upgrades/test_yum_plugins.py

```diff
@@ -89,7 +89,6 @@ def _check_yum_plugins_count(self, client_container_id):
     def _check_package_installed(self, client_container_id, package):
         """Verify if package is installed on docker content host."""
         kwargs = {'host': self.docker_vm}
-        docker_execute_command
```

Comment:
why `docker_execute_command` was removed from the argument?

Reply:
This was hanging without being used, I don't see that func call is doing anything. 

Also, I am not sure why r u calling it as an argument. I believe below this line, this func is being correctly called an argument.

Reply:
my bad I did not notice that.